packages/webapp/src/elections/components/ongoing-election.tsx

```diff
@@ -50,16 +53,11 @@ export const OngoingElection = ({ election }: { election: any }) => {
     if (isError || !memberStats) {
         return <ErrorLoadingElection />;
     }
-    // console.info("currentElection:", currentElection);
-
-    const showNoConsensusInPreviousRoundMessage =
-        (myDelegationSoFar?.length || 0) < election.round;
 
     // TODO: Model all this data to be self-consistent and to abstract the frontend from the complexities of the backend logic
     // start with logged-in user
     // check that they registered for the election (and cast at least one vote?)
     // see if they're still participating
```

Comment:
Anything to amend or remove here? Prob not a bad idea to review comments related to this body of work throughout.